.circleci/config.yml

```diff
@@ -20,27 +29,123 @@ jobs:
     # In many cases you can simplify this from what is generated here.
     # 'See docs on artifact collection here https://circleci.com/docs/2.0/artifacts/'
     - run: mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
-    # This is based on your 1.0 configuration file or project settings
-    - run: add-apt-repository --yes ppa:ubuntu-toolchain-r/test
-    - run: apt-get update
-    - run: apt-get install g++-5 tcl-dev libyaml-cpp-dev libzmq-dev libboost-dev libconfig++-dev libssl-dev libboost-system-dev libboost-test-dev
-    - run: update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-5 1
-    - run: ls; pwd; nproc
-    - run: cd ~/; git clone git@github.com:google/re2.git && cd re2; make && make install
-    - run: cp ~/re2/obj/so/lib* /lib/
-    - run: cd ~/; wget http://tux.rainside.sk/apache/trafficserver/trafficserver-7.1.8.tar.bz2
-    - run: cd ~/; tar xf trafficserver-7.1.8.tar.bz2
-    - run: cd ~/trafficserver-7.1.8 && ./configure --prefix=`pwd`/../ts7 && make -j`nproc` && make install
-    - run: echo "banjax/banjax.so $HOME/ts7/libexec/trafficserver/banjax/" >> ~/ts7/etc/trafficserver/plugin.config
-    - run: echo "regex_map http://^127.0.0.1:8080$/ http://127.0.0.1:8000/" >> ~/ts7/etc/trafficserver/remap.config
-    - run: perl -pi -e 's/CONFIG proxy.config.http.cache.required_headers INT .+/CONFIG proxy.config.http.cache.required_headers INT 0/g' ~/ts7/etc/trafficserver/records.config
-    - run: cat ~/ts7/etc/trafficserver/records.config
-    - run: mkdir build7
-    - run: cd build7 && cmake ../ -DCMAKE_CXX_FLAGS="-I$HOME/ts7/include"
-    - run: cd build7 && make -j3
-    - run: mkdir ~/ts7/libexec/trafficserver/banjax
-    - run: cp config/{auth,solver}.html ~/ts7/libexec/trafficserver/banjax/
-    - run: cp build7/banjax.so ~/ts7/libexec/trafficserver/banjax/
-    # Run tests
-    - run: ./build7/unit-tests
-    - run: python ./test/banjax_behavior_test.py --ts-prefix=$HOME/ts7 --ts-layout="standard"
+
+    - run: |
+        apt-get update -qq &&
+        apt-get upgrade -qq
```

Comment:
I believe the `&&` isn't necessary here. I.e. you can have multiple commands in one `- run` node separated by new lines (and I believe when one fails, the other one is not executed).